crates/control/Cargo.toml

```diff
@@ -19,6 +19,7 @@ futures = "0.3.19"
 hyper = { version = "0.14.16", features = ["full"] }
 once_cell = "1.9.0"
 rand = "0.8.4"
+rusqlite = { version = "*", features = ["bundled", "collation", "column_decltype", "functions", "serde_json", "url"] }
```

Comment:
I didn't want to really decide on how we're querying this stuff just yet. And it seemed like `sqlx` has build-time implications that could add complexity. I'm uncertain about the tradeoffs, given that build databases are treated so differently from something like postgres. I think it could be reasonable to switch to `sqlx` once we actually start writing queries, though. I can add a comment about it, unless you have conviction that `sqlx` is gonna be better. I did just see in the docs that the compile-time query checking is optional.